
# Waxinekaarsje

Als je een waxinekaarsje aan steekt en er een glas overheen zet, zal het kaarsje uit gaan. 
Als je dat waxinekaarsje laat drijven op water terwijl je er het glas er overheen zet, zal het water in het glas stijgen.
Op het internet zwerven er verschillende verklaringen voor dit fenomeen rond.
1. Het kaarsje verbruikt zuurstof, waardoor er een vacuüm ontstaat en het water omhoog wordt gezogen.
2. Het kaarsje verwarmt de lucht in het glas, waardoor de lucht uitzet. Wanneer de kaars dooft koelt de lucht af waardoor de druk afneemt en er water omhoog wordt gezogen.
3. Bij het verbranden van de kaars ontstaan er waterdamp en koolstofdioxide. De waterdamp condenseert aan de binnenkant van het glas, waardoor er minder gas in het glas is en de druk afneemt. Hierdoor wordt het water omhoog gezogen.

```{exercise}
Onderzoek welke verklaring het meest waarschijnlijk is en leg uit waarom de andere verklaringen minder waarschijnlijk zijn.
```

Wij denken dat het antwoord nr.2 is. Wij denken dit doordat het waxinelapje veel stijgt als hij uitgaat. We hebben dit getest door het glas op te warmen met de aansteker en daarna in het water te zetten en hetzelfde gebeurde. Hieronder fotobewijs:


In [ ]:
import math

def lucht_temperatuur(V_water_liter, diameter_cm, V_lucht_liter, T_init_C=20):
    
    # Constanten
    rho = 1000          # kg/m^3, dichtheid water
    g = 9.81            # m/s^2, zwaartekracht
    p_atm = 101325      # Pa, atmosferische druk

    # Conversies
    V_water = V_water_liter / 1000       # liter -> m³
    V_lucht = V_lucht_liter / 1000       # liter -> m³
    r = diameter_cm / 2 / 100             # cm -> m

    # Buisoppervlakte
    A = math.pi * r**2

    # Waterstijging
    delta_h = V_water / A

    # Drukverhoging door waterkolom
    delta_p = rho * g * delta_h
    p2 = p_atm + delta_p

    # Volume van lucht na waterverplaatsing
    V2 = V_lucht - V_water

    # Temperatuur na verwarming (K)
    T1_K = T_init_C + 273
    T2_K = T1_K * (p2 * V2) / (p_atm * V_lucht)

    # Terug naar °C
    T2_C = T2_K - 273

    return T2_C, delta_h

# gegevens: 50 mL water, buis 4.15 cm, luchtvolume 1 L
T2, stijging = lucht_temperatuur(0.05, 4.15, 1)
print(f"Waterstijging: {stijging*100:.2f} cm")
print(f"Luchttemperatuur: {T2:.2f} °C")


Waterstijging: 3.70 cm
Luchttemperatuur: 6.35 °C


wat je hier uit de code kan halen is dat voor de gemeten stijging van 50 ml(na het warmen van de buis), volgens de iedeale gaswet de temperatuur van de buis 6.35 graden opgewarmd moet zijn. In het experiment is de lucht in de buis voor het plaatsen van de buis in het water opgewarmd met een aantal graden, waardoor je uit de ideale gaswet kan concluderen (zoals is laten zien) dat het opstijgen van het water vooral met de temperatuur van de lucht te maken heeft, dus is optie 2 de belangrijkste reden voor het stijgen van het water. 